packages/eslint-config-sentry-react/index.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     'getsentry/jsx-needs-il8n': ['off'],
     'testing-library/render-result-naming-convention': 'off',
     'testing-library/no-unnecessary-act': 'off',
+    'typescript-sort-keys/interface': ['error', 'asc', {'caseSensitive': true, 'natural': false, 'requiredFirst': true }],
```

Comment:
Does `interface` include type aliases as well?

Reply:
Yes, there's also one for enums, but I didn't turn that one on yet.